integration-tests/test-sequencer.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('lodash/fp')
+const Sequencer = require('@jest/test-sequencer').default
+
+
+// Start these tests first since they are long-running.
+// This is most relevant on CI, where the number of simultaneous tests is capped.
+const testsToRunFirst = [
+  'create-interactive-analysis',
+  'run-workflow',
+  'run-workflow-on-snapshot'
+]
```

Comment:
`jest/1-run-workflow.integration-test.js` had a comment:

```
/*
 * File named with "1-" to make Jest start it first, since it's a long-running test.
 * This is most relevant on CI, where the number of simultaneous tests is capped.
 */
```

Not sure if that actually worked, since it didn't look like Jest was applying any order to tests by default. But these tests will now be run first.
https://jestjs.io/docs/configuration#testsequencer-string

Reply:
Oh, which tests to run was controlled by CircleCI, not Jest.

https://github.com/DataBiosphere/terra-ui/blob/a53c5252cbacbe99ee885bec05f46444e5c0ac39/.circleci/config.yml#L173-L174

Reply:
Nice, CircleCI was helpful in figuring out what should be passed to `circleci tests split`.

https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/9710/workflows/e9c4821a-d1e4-4203-ac1b-53c4cb4f5000/jobs/43255

```
Error autodetecting timing type, falling back to weighting by name. Autodetect no matching filename or classname.  If file names are used, double check paths for absolute vs relative.
Example input file: "/mnt/ramdisk/integration-tests/tests/analysis-context-bar.js"
Example file from timings: "jest/1-create-interactive-analysis.integration-test.js"
```

Reply:
Actually, I guess absolute vs relative path doesn't make a difference here. This failing on the first run is just a consequence of renaming the test files.

Reply:
If CircleCI is splitting tests by timing, I don't think we need to try to control the order of tests. Removed the custom sequencer.